hasura/metadata/actions.yaml

```diff
@@ -111,6 +112,7 @@ actions:
         - name: verification_key
           value_from_env: HASURA_EVENT_SECRET
     permissions:
+      - role: api-user
```

Comment:
Unless for other api users, for the bot we don't need the `deleteCircle`

Reply:
yeah, trying to see if we should activate others here.